### PR TITLE
Add larva init command

### PR DESCRIPTION
**Pull request by laras126**
* Add an `init` script for the `larva` binary that copies a directory with a scaffold of the larva architecture.
* Clean up some logic in writeJson and server scripts
* Update larva package docs to outline workflow for using the script. These will need to be tested once the code is available on npm. Since the package is released under an `-alpha` tag, this is an acceptable workflow.

### CODE REVIEW

File: packages/larva/scripts/larva/larva.config.js [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+		aliases: {
+			'@js': path.resolve( './src/js' )
+		},
+		entries: {
+			common: './entries/common.entry.js'
+		}
+	},
+
+	backstop: {
+		// testBaseUrl: 'https://notlaura.com',
```
Comment (sarahsmo) [inline packages/larva/scripts/larva/larva.config.js]: Do we still need this comment in here?

File: packages/larva/scripts/larva/larva.config.js [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+		larvaPatternsDir: path.resolve( './node_modules/@penskemediacorp/larva-patterns' ),
+		projectPatternsDir: path.resolve( './src/patterns' ),
+		ignoredModules: [
+			'footer-menus',
+			'footer-social',
+			'footer-tip'
+		]
+	},
+
+	parser: {
```
Comment (sarahsmo) [inline packages/larva/scripts/larva/larva.config.js]: we want this here even though has no values?

File: packages/larva/scripts/larva/README.md [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+
+* `npm run lint` - Lint SCSS and JS in src/
+* `npm run lint:scss` - Lint SCSS.
+* `npm run lint:js` - Lint JS in src/.
+* `npm run lint-fix:scss` - Lint and fix auto-fixable errors in SCSS.
+* `npm run lint-fix:js` - Lint and fix auto-fixable errors in JS.
+
+## Read the docs
+
+Read about developing UI [on the Larva Wiki](https://github.com/penske-media-corp/pmc-larva/wiki).
```
Comment (sarahsmo) [inline packages/larva/scripts/larva/README.md]: YAY LARVA WIKI!!

File: packages/larva/scripts/larva/larva.config.js (hunk 1 above)
Comment (laras126) [reply packages/larva/scripts/larva/larva.config.js]: Yeah - this is for toggling using a remote URL vs. the localhost – I need to document/organize this. Will do in later PR. Thanks!

File: packages/larva/scripts/larva/larva.config.js (hunk 2 above)
Comment (laras126) [reply packages/larva/scripts/larva/larva.config.js]: I need to add a default value for the parser I think - need to test it. Also will do in a clean up PR.

**Pull request merged**